lxd/container_lxc.go

```diff
@@ -6012,8 +6012,15 @@ func (c *containerLXC) idmapsetFromConfig(k string) (*shared.IdmapSet, error) {
 }
 
 func (c *containerLXC) NextIdmapSet() (*shared.IdmapSet, error) {
```

Comment:
The check in init will trigger first and gives a much clearer error, so I think it's worth keeping.